armi/reactor/blueprints/__init__.py

```diff
@@ -321,18 +324,80 @@ def _prepConstruction(self, cs):
                 self.assemblies[aDesign.name] = a
             if currentCount != 0:
                 assemblies.setAssemNumCounter(currentCount)
-
-            self._checkAssemblyAreaConsistency(cs)
-
             runLog.header("=========== Verifying Assembly Configurations ===========")
+            self._checkAssemblyAreaConsistency(cs)
 
-            # pylint: disable=no-member
-            getPluginManagerOrFail().hook.afterConstructionOfAssemblies(
-                assemblies=self.assemblies.values(), cs=cs
+            # if assemblies are defined in blueprints, handle meshing
+            # assume finest mesh is reference
+            assemsByNumBlocks = sorted(
```

Comment:
@john-science, this is the only code called under all circumstances, and its not particularly slow. the other 2 calls depend on the settings enabled. Stuff should get faster for basically all use cases since before if we had a core with 50 assemblies and 5 assemblies in blueprints it would

1) make the 5 unique assemblies in blueprints
2) deepcopy 50 more to populate the core
3) axially expend cold to hot the 5 assemblies in blueprints + 50 in core for (slow thing 55 total times)

now its
1) make the 5 unique assemblies in blueprints
2) axially expand those 5 assemblies (slow thing 5 total times)
2) deepcopy already expanded assemblies 50 more to populate the core